synapse_token_authenticator/utils.py

```diff
@@ -1,5 +1,7 @@
-import requests
+from base64 import b64encode
 from urllib.parse import urljoin
+
+import requests
```

Comment:
There is still one requests call. If that can't be changed to the builtin client, could you document why? (A comment or so)

Reply:
Totally missed this one, thx

Reply:
Huh, this function seems to be unused

https://github.com/famedly/synapse-token-authenticator/blob/ba1c267f4d7e449ce3a28a68ec1ded25a9ce3e53/synapse_token_authenticator/utils.py#L27-L34

Should we remove it?